rbi/stdlib/optparser.rbi

```diff
@@ -0,0 +1,778 @@
+# typed: __STDLIB_INTERNAL
+class OptionParser
+  sig { params(to: T.untyped, name: T.untyped).returns(T.untyped) }
```

Comment:
Our current `rbi`'s use a minorly different whitespacing:
```
  sig {returns(Integer)}
```
Note lack of spaces in curlies. Would you be up to make your rbi's  consistent with existing ones?